src/main/java/com/example/carsharing/controller/CarController.java

```diff
@@ -29,9 +29,9 @@ public class CarController {
     public CarResponseDto add(@RequestBody CarRequestDto carRequestDto) {
         return carMapper.mapToDto(carService.add(carMapper.mapToEntity(carRequestDto)));
     }
-    // GET /cars
+    // GET /cars/
 
-    @GetMapping
+    @GetMapping("/")

```

Comment:
I'm not sure about it. Is URL for getting all cars .../cars/ or not? 
Better use simple variant like .../cars

Reply:
I was just test for solving swagger problem. I fixed that in other branch